src/coreclr/nativeaot/Runtime.Base/src/System/Runtime/ExceptionHandling.cs

```diff
@@ -609,8 +609,11 @@ private static void DispatchEx(scoped ref StackFrameIterator frameIter, ref ExIn
 
             byte* prevControlPC = null;
             byte* prevOriginalPC = null;
+            UIntPtr prevControlSP = MaxSP;
             UIntPtr prevFramePtr = UIntPtr.Zero;
             bool unwoundReversePInvoke = false;
+            IntPtr pReversePinvokePropagationCallback = IntPtr.Zero;
```

Comment:
```suggestion
            IntPtr pReversePInvokePropagationCallback = IntPtr.Zero;
```
Nit: The previous line uses `PInvoke` casing. It would be nice to be consistent.